### PR TITLE
Schema added for posts, Addition,Deletion,Retrieval,Updation and Automatic Scheduled Posting Feature added

### PR DESCRIPTION
**Pull request by TG1999**
Signed-off-by: tushar goel <tushar.goel.dav@gmail.com>

### CODE REVIEW

**Comment by TG1999** [comment]
#207 Schema added

File: server/schema/posts.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  DownVotes:{
+    type:Object
+  },
+  tags:{
+    type:Array
+  },
+  Comments:{
+      type:[{user:ObjectId,
+      content:String}]
+  },
```
Comment (vaibhavdaren) [inline server/schema/posts.js]: https://mongoosejs.com/docs/guide.html#timestamps

Comment (vaibhavdaren) [reply server/schema/posts.js]: why aren't we using the default timestamp class? what will be the tradeoffs?
Lets research and change them elsewhere too.

Comment (TG1999) [reply server/schema/posts.js]: Okay got it

Comment (TG1999) [reply server/schema/posts.js]: Resolved this issue in next commit

**Comment by TG1999** [comment]
#207 addition of posts done

**Comment by TG1999** [comment]
#207 Automatic Posting Feature added

**Comment by TG1999** [comment]
#207 Updation of Post feature added

**Comment by vaibhavdaren** [comment]
Why are merge and other commits present in your pull request?
Only one commit should be present.

Correct use of git:-
1) pull the latest version from dev/gsoc
2) make your changes
3) squash you local commits into one.
4) send pr

**Comment by TG1999** [comment]
I have made multiple commits to maintain the history of commits since it was not a single feature, their were multiple features

**Comment by TG1999** [comment]
So that if one step is wrong in my further commits I can revert from their and can make new commits for it

**Pull request closed without merging**